### ISSUE TITLE
Missing pagination option for GET purchase orders API

### ISSUE THREAD

**Issue by stansv** [XeroAPI/xero-node#266]
Hi! Looks like there's no PagingArgs type supported in request options..

I use `3.0.21`.

https://developer.xero.com/documentation/api/purchase-orders

**Comment by getDorin** [comment]
Ran into this issue today as well, trying to hack around it but it looks like this feature just isn't there.
Would be nice to have it as it is recommended to fetch with pagination etc.

**Comment by getDorin** [comment]
Hi I stand corrected, actually you can request pagination using the Xero-Node package. Just pass the page in the params object like this: 
```
yield xeroInstance.core.invoices.getInvoices({
    params: { page: pagination }
});
```

Hope this mmight help someone as this example is not included in the docs.

**Comment by RettBehrens** [comment]
A quick note on why we’re closing this issue:

With the release of version 4.x of the Xero-Node SDK in December of 2019, we delivered a complete overhaul of the SDK. 

We moved from writing our SDKs by hand to generating our SDKs using our [Xero-OpenAPI](https://github.com/XeroAPI/Xero-OpenAPI) specs. By utilizing Xero-OpenAPI specs to generate code we’ve improved the developer experience of maintaining the SDK and improved the accessibility for our developer community to contribute via pull requests to the Xero-OpenAPI specs. 

We also built our new SDKs to use the industry standard OAuth2 in preparation for our March 2021 deprecation of OAuth1. For more details, see our [blog post on our decision to move to OAuth2](https://devblog.xero.com/an-update-on-why-we-are-saying-goodbye-oauth-1-0a-hello-oauth-2-0-6a839230908f). 

Version 3.x using Oauth1 and version 4.x using OAuth2 do not share any code as we now generate our SDKS. Going forward we’re focusing on building, maintaining and improving our generated SDKs but you are welcome to fork the [‘oauth1’ branch](https://github.com/XeroAPI/xero-node/tree/oauth1) and apply changes if you wish to implement something for your integration.
